deploy/examples/multus-validation.yaml

```diff
@@ -1,5 +1,5 @@
 ####################################################################################################
-# This manifest contains a Kubernetes Job and supporting definitions for running Rook's Multus
+# This manifest contains a Kubernetes job and supporting definitions for running Rook's Multus
```

Comment:
This is intentional. Please revert.

Reply:
@BlaineEXE  wrote:
> This is intentional. Please revert.

done. commit removed